src/components/Pagination/pagination.scss

```diff
@@ -47,21 +47,39 @@
     height: 32px;
     color: $color-primary-dark;
     background-color: transparent;
-    border: 0;
     border-radius: $base-border-radius;
     cursor: pointer;
 
-    &:not(:disabled):hover {
-      color: $color-primary;
+    &:not(&--withLabel) {
+      border: 0;
+    }
+
+    &:not(:disabled):not(&--withLabel) {
+      &:hover {
+        color: $color-primary;
+      }
     }
 
     &--disabled {
       color: $sb-dark-blue-25;
       cursor: not-allowed;
     }
 
-    &:hover {
-      background-color: $light-50;
+    &--withLabel {
```

Comment:
this is not a proper css class, it should be `with-label`

Reply:
Good catch! Thanks @lisilinhart 
 It's an old habit I am struggling to get rid of 😄 We had an internal _(BEM like)_ naming convention in my previous job.
 
 I pushed the change ✅